app/src/main/java/org/groundplatform/android/ui/map/gms/features/LineStringRenderer.kt

```diff
@@ -43,6 +56,7 @@ constructor(private val resources: Resources, private val bitmapUtil: BitmapUtil
     val bitmap = bitmapUtil.fromVector(R.drawable.ic_circle_marker)
     CustomCap(BitmapDescriptorFactory.fromBitmap(bitmap))
   }
+  private var distanceMarker: Marker? = null
```

Comment:
Same here - "Distance marker" is specific to one particular use of LineStrings. If we were thinking in terms of generic linestrings, we could cll this the "tooltipMarker".